osquery/tables/system/posix/python_packages.cpp

```diff
@@ -0,0 +1,119 @@
+/*
+*  Copyright (c) 2014-present, Facebook, Inc.
+*  All rights reserved.
+*
+*  This source code is licensed under the BSD-style license found in the
+*  LICENSE file in the root directory of this source tree. An additional grant
+*  of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#include <string>
+#include <fstream>
+
+#include <boost/algorithm/string/trim.hpp>
+
+#include <osquery/core/conversions.h>
+#include <osquery/filesystem.h>
+#include <osquery/logger.h>
+#include <osquery/system.h>
+#include <osquery/tables.h>
+
+
+namespace osquery {
+namespace tables {
+
+const int kNumFields = 2;
+const std::set<std::string> kPythonPath = {
+  "/usr/local/lib/python2.7/", "/usr/lib/python2.7/dist-packages",
```

Comment:
For these directories, let's just point it at the explicit parent directory which contains the egg files. I believe I might've encouraged you to use the top level `/usr/local/lib/python2.7` directory, but my thought process there was you could point your parser to `/usr/local/lib/python2.7`, and it would grab _all_ sub-directories. Similarly, if someone wanted to add a `/usr/local/lib/python3.5` or `/usr/lib/python2.7`, all they'd need to do would be to add the directory to this set, and it'd auto-magically be picked up and parsed. However, doing this would likely require two levels of traversal on your part, and is likely not worth the extra logic. I'd say instead, let's change this to be:
```
const std::set<std::string> kPythonPath = {
  "/usr/local/lib/python2.7/site-packages", "/usr/lib/python2.7/dist-packages",
}
```
And then we can avoid needing the `checkOverlap` function down below.